externs/shaka/player.js

```diff
@@ -827,16 +827,18 @@ shaka.extern.ProducerReferenceTime;
  *   state, e.g., for persistent license storage.
  *   <br>
  *   Defaults to <code>false</code>.
- * @property {string} videoRobustness
- *   A key-system-specific string that specifies a required security level for
- *   video.
- *   <br>
- *   Defaults to <code>''</code>, i.e., no specific robustness required.
- * @property {string} audioRobustness
- *   A key-system-specific string that specifies a required security level for
- *   audio.
- *   <br>
- *   Defaults to <code>''</code>, i.e., no specific robustness required.
+ * @property {Array.<string>} videoRobustness
+ *   A key-system-specific Array of strings that specifies a required security
+ *   level for video. Can be an array of strings to represent multiple
+ *   robustness.
+ *   <br>
+ *   Defaults to <code>['']</code>, i.e., no specific robustness required.
```

Comment:
I feel like the default for these two should be an empty array, instead of an array containing an empty string. As-is it looks like if an empty array is passed in it will cause the drm infos to all be removed.

Reply:
I guess `[]` and `['']` should result in the same behavior?